Lib/test/test_email/test_email.py

```diff
@@ -5541,7 +5542,11 @@ def test_long_headers_flatten(self):
         result = fp.getvalue()
         self._signed_parts_eq(original, result)
 
-
+class TestHeaderRegistry(TestEmailBase):
+    # See issue gh-93010.
+    def test_HeaderRegistry(self):
+        reg = HeaderRegistry()
+        reg('Content-Disposition', 'attachment; 0*00="foo"')
```

Comment:
Shouldn't there be a test that the expect defect shows up in the defect list?

Reply:
Sure! I have added a check for this. Does that work?